fidesctl/src/fidesctl/core/system.py

```diff
@@ -64,13 +64,95 @@ def generate_redshift_systems() -> List[System]:
     return redshift_systems
 
 
+def describe_rds_clusters() -> Dict[str, List[Dict]]:
+    """
+    Creates boto3 rds client and returns describe_db_clusters response.
+    """
+    rds_client = boto3.client(
+        "rds",
+    )
+    describe_clusters = rds_client.describe_db_clusters()
+    return describe_clusters
+
+
+def describe_rds_instances() -> Dict[str, List[Dict]]:
+    """
+    Creates boto3 rds client and returns describe_db_instances response.
+    """
+    rds_client = boto3.client(
+        "rds",
+    )
+    describe_instances = rds_client.describe_db_instances()
+    return describe_instances
+
+
+def transform_rds_systems(
+    describe_clusters: Dict[str, List[Dict]], describe_instances: Dict[str, List[Dict]]
+) -> List[System]:
+    """
+    Given a describe clusters and describe instances responses, build a system object
```

Comment:
```suggestion
    Given "describe_clusters" and "describe_instances" responses, build a system object
```

Reply:
basically just wanted to make it more obvious that it was related to a function/code and not just English since it looks off

Reply:
done